docsource/modules150-160.rst

```diff
@@ -552,7 +552,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | payment_buckaroo                                |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |new| payment_custom                            |                      |                                                 |
+| |new| payment_custom                            |                      |Renamed from payment_custom.                     |
```

Comment:
```suggestion
| |new| payment_custom                            |                      |Renamed from payment_transfer.                     |
```

I guess...

Reply:
It should include the word `Done` for getting it tested in the CI:
 
```suggestion
| |new| payment_custom                            |                      | Done. Renamed from payment_custom.               |
```

Reply:
The second column (status) is parsed, not the third(extra description) 
Except that, you're right.

Reply:
Done, thank you :+1: